python/tk_desktop/command_panel/recent_list.py

```diff
@@ -22,7 +22,7 @@ class RecentList(BaseIconList):
 
     def __init__(self, parent):
         super(RecentList, self).__init__(parent, QtGui.QHBoxLayout())
-        self._layout.addStretch(1)
```

Comment:
This line was what was stopping the buttons from stretching out as the window expanded.